tests/extension_tests/deferred_bindings_tests/deferred_bindings_functions/deferred_bindings_enabled/function_app.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+import azure.functions as func
+import azurefunctions.extensions.bindings.blob as bindings
```

Comment:
as blob or funcblob or something - this can become base for the new sampes.

Reply:
Renamed to blob. Good point. It's imported as "blob" in the extension repo samples, so now it's consistent